### ISSUE TITLE
Consumer Count Not displaying

### ISSUE THREAD

**Issue by Saravana1402** [obsidiandynamics/kafdrop#392]
Hello Everyone,

My application is currently being tested. But I can't see the consumer group count in kafdrop between tests. Can somebody help me solve this problem?

![consumer](https://user-images.githubusercontent.com/34307939/175022872-83b6f226-913d-4302-87e9-e559329b33ba.png)

**Comment by davideicardi** [comment]
It could be related to #374 ?
Which version are you using?

**Comment by Saravana1402** [comment]
Version: kafdrop-3.28.0-SNAPSHOT

**Comment by davideicardi** [comment]
Can you try with kafdrop-3.31.0-SNAPSHOT ?

**Comment by Saravana1402** [comment]
I had tested with the latest version. But still facing the same issue. I saw one error log while running the kafdrop.  The jar is not able to fetch or take the data from the consumer group.

2022-06-24 09:44:40.921  INFO 3688 [  XNIO-1 task-2] k.c.BasicErrorController                 : errorAtts: {timestamp=2022-06-24T09:44:40.921+0000, status=404, error=Not Found, message=Not Found, path=/topic/m2m-all-predecoder-topic/consumer/m2m-topic-consumer-group}
